pkg/cmd/attestation/verification/sigstore.go

```diff
@@ -150,14 +150,9 @@ func newCustomVerifier(trustedRootFilePath string) (*verify.SignedEntityVerifier
 	}
 
 	verifierConfig := []verify.VerifierOption{}
-	verifierConfig = append(verifierConfig, verify.WithSignedCertificateTimestamps(1))
```

Comment:
Sigstore's public good Fulcio uses SCTs, but GitHub's Sigstore Fulcio does not, so this prevents us from supplying a GitHub-compatible `--custom-trusted-root`.

I believe it's okay to remove this check, because we are leaving the more inclusive `WithObserverTimestamps` which could be SCTs or signed timestamps.

Reply:
Thanks for including this explanation. Maybe it's worth adding a comment noting that `verify.WithObserverTimestamps` covers both cases?